examples/cloudml-document-ai-patents/automl_image.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Runs AutoML Image classifier, writes the results to BigQuery and copies doc to GCS."""
+
+import os
+import sys
+import utils
+import logging
+
+from google.cloud import storage, bigquery, automl
+
+# TODO: Clean classify write into submodules + handle initialization of the clients better (is it the right place).
+# TODO: Rename sample_handler + is it necessary to feed the string to the API (vs path)
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
only the files intended for execution should set the log level, remove this line

Reply:
this should be fixed in every file that isn't intended to be directly executed.